### PR TITLE
docs(#1001): multi-file-input field only displayed in full size

### LINKED ISSUE #1001: Dokumenten-Feld nur in Größe 12 anschaulich
https://git.muenchen.de/digitalisierung/digiwf-support/-/issues/157

**Acceptance Criteria**

- [ ] Benutzung von Dokumenten-Feld in Bezug auf Gruppierung ist in DigiWF-Doku beschrieben

### PR DESCRIPTION
**Pull request by lmoesle**
### Description

Short description or comments

### Reference

Issues: closes #1001

### Check-List

- [ ] All Acceptance criteria of user story are met
- [ ] JUnit tests are written (60% CodeCov)
- [ ] [Internal Review]([https://github.com/it-at-m/digiwf-core/blob/dev/CHANGELOG.md](https://confluence.muenchen.de/display/MPdZ/Review+-+DigiWF)) is maintained
- [ ] Documentations [external](https://github.com/it-at-m/digiwf-core/tree/dev/docs) and [internal](https://wiki.muenchen.de/betriebshandbuch/index.php?title=DigiWF&sfr=betriebshandbuch) are completed
- [ ] Smoketest successful (Manual E2E-Test depending on Change) 
- [ ] No Branch waste left
- [ ] [Board](https://app.zenhub.com/workspaces/digiwf-621f70bf50ea1100120b7e93/board) is up-to-date
- [ ] Openshift environments are prepared (Secrets, etc.) and release-issue is maintained

### CODE REVIEW

**Review by lehju — APPROVED** [review]

File: docs/src/modeling/forms/fields/README.md [hunk 1]
```diff
@@ -186,20 +186,31 @@ Es gibt lediglich einen Unterschied bei der Auswahl and Anzeigemöglichkeiten.
...
 ![Formular Editor](~@source/modeling/forms/fields/files-context.png)
 
-::: tip
-Je nachdem ob es sich um ein Start- oder Benutzerformular handelt, muss der Zugriff zusätzlich konfiguriert werden.
-Dies kann in der jeweiligen Dokumentation nachgelesen werden
+#### Konfiguration
+
+Das Datei-Upload-Feld benötigt eine zusätzliche Konfiguration im Usertask des Prozesses, damit Dateien gespeichert und gelesen werden können.
+
+In den Inputparameter `app_file_paths_readonly` und `app_file_paths` müssen im Usertask der/die Ordner angegeben werden, auf die der User zugreifen darf.
```
Comment (darenegade) [inline docs/src/modeling/forms/fields/README.md]: Das ist vielleicht etwas verwirrend, da in der CoCreation die so nicht heißen:

image

**Review by darenegade — CHANGES_REQUESTED** [review]
Sonst passt es

**Review by darenegade — APPROVED** [review]
Passt. Danke

File: docs/src/modeling/forms/fields/README.md (hunk 1 above)
Comment (lmoesle) [reply docs/src/modeling/forms/fields/README.md]: Danke, ich habe die 2 Labels auch noch dazu geschrieben

**Pull request merged**